src/drake/fs.clj

```diff
@@ -16,6 +16,19 @@
 (def drake-ignore "Names of files or directories to be ignored by Drake"
   #{"_logs"})
 
```

Comment:
I did the loop to be defensive. It's unnecessary but it's good practice. What if ```.getParentFile``` has a bug on some weird file system?

You're right about the nil, I'll use a when-let here.